src/Jellyfin.Maui/Models/UserStateModel.cs

```diff
@@ -6,37 +6,22 @@
 public class UserStateModel
 {
     /// <summary>
-    /// Initializes a new instance of the <see cref="UserStateModel"/> class.
+    /// Gets or sets the user id.
```

Comment:
```suggestion
    /// Gets the user id.
```

`init` setters should not be part of the documentation